src/main/java/com/bynder/sdk/model/HttpConnectionSettings.java

```diff
@@ -28,6 +29,11 @@
      * Retry connection in case it fails the first time.
      */
     private final boolean retryOnConnectionFailure;
+    /**
+     * Whether or not to enable a {@link HttpLoggingInterceptor} on the http client used, 
```

Comment:
can you change http to HTTP? in order to to make it consistent with the rest of the javadocs in the sdk.